.gitlab-ci.yml

```
@@ -131,6 +136,17 @@ test:static:release:
   variables:
     BUILD_TYPE: Release
 
+trigger:mender-mcu-integration:
+  rules:
+  - if: $CI_COMMIT_BRANCH =~ /^(pr_[0-9]+)$/
```

```suggestion
  - if: $CI_COMMIT_BRANCH =~ /^pr_[0-9]+$/
```

:point_up: we don't need the grouping

---

Am I doing something wrong with the branch? It can't find the mender-mcu-integration reference now